app/src/main/java/com/example/campusguide/directions/GetDirectionsDialogFragment.kt

```diff
@@ -70,9 +80,10 @@ class GetDirectionsDialogFragment constructor(private val options: DirectionsDia
     }
 
     private fun setDefaultText(view: View, editTextId: Int, text: String?) {
-        if (text != null) {
+        if (text != null) { //Using current location for directions
```

Comment:
Remove comment, in the future the text might have been set because of something other than current location